config/locales/views.en.yml

```
@@ -92,6 +92,16 @@ en:
       delete:
         action: "Delete my account"
         heading: "Unsatisfied?"
+    blocks:
+      blocked: "blocked %{time} ago"
+      none: "No one has been blocked yet"
```

this reads a bit too sterile, what about:

```suggestion
      none: "You are not blocking anyone."
```